src/components/navbar/navbar.js

```diff
@@ -3,31 +3,42 @@ import Link from 'gatsby-link';
 import SideMenu from '../SideMenu';
 
 class Navbar extends Component {
-  constructor(props) {
-    super(props);
-  }
+    constructor(props) {
+        super(props);
+    }
 
-  render() {
-    const { links } = this.props;
-    const getLinks = links.map((link, index) => {
-      let filePath = `/${link.section}/${link.file}`;
-      return (
-        <li key={index}><Link to={filePath} className="navigation__list-item">{link.name}</Link></li>
-      );
-    });
+    render() {
+        const { links } = this.props;
+        const getLinks = links.map((link, index) => {
+            console.log(link.url);
 
-    return (
-      <nav className="navigation">
-        <Link to="/" className="navigation__logo"></Link>
-        <ul>
-          {getLinks}
-        </ul>
-        <div className="navigation__sidemenu">
-          <SideMenu />
-        </div>
-      </nav >
-    );
-  }
+            return link.url.substring(0, 4) === 'http'
+                ?
+                <li key={index}>
+                    <a href={link.url} className="navigation__list-item">
+                        {link.name}
+                    </a>
+                </li>
+                :
+                <li key={index}>
+                    <Link to={link.url} className="navigation__list-item">
+                        {link.name}
+                    </Link>
+                </li>
+        });
+
+        return (
+            <nav className="navigation">
+                <Link to="/frc" className="navigation__logo"></Link>
+                <ul>
+                    {getLinks}
+                </ul>
+                <div className="navigation__sidemenu">
+                    <SideMenu />
+                </div>
+            </nav>
+        );
+    }
```

Comment:
@dylankfernandes I don't see any reason not to do the map() inline here. I think it's more legible than assigning it to a variable, which moves the meat of this component above the rest of it.